lib/renderer.coffee

```diff
@@ -25,6 +25,10 @@ exports.toHtml = (text='', filePath) ->
 exports.toRawHtml = (text='', filePath) ->
   render text, filePath
 
+exports.getBlockId = (bufferRowPosition) ->
+  if exports.blocksPositions
```

Comment:
I know but everything else has failed:

 * `@blocksPositions`
 * Declare `blockPositions` at the top of the file

I also try to restructure the code but blocks positions are produced in the async task `worker.coffee`. The only way I found to retrieve this information was to use an event/listener. Is there another way ?